pkg/snapshot/iscsi.go

```diff
@@ -50,6 +50,8 @@ const (
 	// Naa preffix for loopback devices in configfs
 	// for example snID 128, the loopback device config in /sys/kernel/config/target/loopback/naa.1990000000000128
 	obdLoopNaaPreffix = 199
+
+	obdMaxDataAreaMB = 4
```

Comment:
what is it used for?

Reply:
It is used for setting the max_data_area_mb param for tcmu devices, which is the mmap buffer size for I/O requests. The default max_data_area_mb value is 1024MB which is too big and may cause memory waste.

Reply:
could you add the comment for the variable? I think it is good for the reader.

Reply:
yes, a description is added.